feat: use the method parameter name if annotation configuration is not provided

aureamunoz opened the pull request:
if no annotation values (for "name" or "value") are supplied the method parameter name will be used
Related to Quarkus [#3992](https://github.com/quarkusio/quarkus/issues/3992) issue

Could you @geoand please review this and forward it to someone else if necessary?

geoand commented:
This looks good to me!

@asoldano WDYT?

geoand commented:
I believe it should be easy @asoldano.

@aureamunoz I assume all that needs to be done is to add the maven compiler configuration to the spring-web integration test and change of or more of the Controllers to showcase the feature?

aureamunoz commented:
yes, of course. I added the mvn compiler config, a few new tests and modified a few for showcasing this feature. I let you take a look and give me feedback. @geoand @asoldano

geoand commented:
Assuming the tests pass, this looks great!

geoand commented:
@asoldano the failures seem unrelated. Could you restart CI please?

geoand commented:
Seems like CI is green now :)

asoldano commented:
Merged, thanks!

geoand commented:
Thanks @asoldano !
